gradio/blocks.py

```diff
@@ -205,8 +205,10 @@ def update(**kwargs) -> dict:
     Updates component parameters.
     This is a shorthand for using the update method on a component.
     For example, rather than using gr.Number.update(...) you can just use gr.update(...).
+    Note that your editor's autocompletion will suggest proper parameters
+    if you use the update method on the component.
 
-    Demos: blocks_update, blocks_essay_update
+    Demos: blocks_update, blocks_essay, blocks_essay_update
```

Comment:
Good job!

Would prefer if the default example is Component.update version.
```suggestion
    Demos: blocks_essay, blocks_update, blocks_essay_update
```